### ISSUE TITLE
Scrolling in Deezer playlists very slow and causing 100% CPU usage one 1 core.

### ISSUE THREAD

**Issue by wayneyooktz** [tiliado/nuvolaplayer#233]
Hello,

as stated above, scrolling makes the app basically unusable.
When scrolling, "/usr/lib/x86_64-linux-gnu/webkit2gtk-4.0/WebKitWebProcess" goes to 100 %CPU in htop. Behaviour is 100% reproducable, occurs *every time* I'm running the program.

**Distro:** Ubuntu GNOME 16.04 (up-to-date), with 4.4.0-23-generic
**libwebkit2gtk-4.0-37** Version: 2.10.9-1ubuntu1
**libwebkit2gtk-4.0-37-gtk2** Version: 2.10.9-1ubuntu1
**nuvolaplayer3** Version: 3.0.2-0+1~xenial

I have again attached the screenshot I took yesterday.
If any further information is required, let me know.

Best regards.

![bildschirmfoto von 2016-05-18 15-04-01](https://cloud.githubusercontent.com/assets/19466760/15389981/b2013846-1db9-11e6-8707-9c40fc37cb7d.png)

**Comment by fenryxo** [comment]
Hello @wayneyooktz. Thanks for taking the time to report this issue. Is it reproducible also in [the WebKitGtk MiniBrowser](https://github.com/tiliado/nuvolaplayer/wiki/Bug-Reporting-Guidelines#webkitgtk-minibrowser)?

**Comment by wayneyooktz** [comment]
Hello @fenryxo. No, performance in Mini Browser is fine. Opening sites, scrolling and everything else is as smooth as one would expect.

**Comment by fenryxo** [comment]
> Opening sites, scrolling and everything else is as smooth as one would expect.

You have mentioned "sites" without specifying which ones. Have also tried the Deezer page that causes trouble in Nuvola?

**Comment by wayneyooktz** [comment]
Sorry, for whatever reason I did not take that into consideration. I just opened deezer.com with Mini Browser, logged in and had the same issue as with nuvolaplayer. While doing that, I repeatedly get a file not found error for "libvdpau_va_gl.so", though I'm not sure if that has anything to do with it.

**Comment by fenryxo** [comment]
Since the issue is reproducible also in the MiniBrowser, I believe the bug is not in Nuvola Player code. Let's investigate further ;-)

1. Could you post a complete debugging output as specified in [Bug Reporting Guidelines](https://github.com/tiliado/nuvolaplayer/wiki/Bug-Reporting-Guidelines#debugging-output). There may be some clues.
2. Regarding the "not found error for libvdpau_va_gl.so", do you have the `libvdpau-va-gl` package installed? If not, could you install it and report back whether it makes any difference?

**Comment by wayneyooktz** [comment]
Okay, the log can be found [here](https://gist.github.com/wayneyooktz/a22a334b01607f08cb27c36ae281fce8).

And a package named exactly `libvdpau-va-gl` can't be found in the Ubuntu repo - however there is one with a **1** at the end - so `libvdpau-va-gl1` - and installing that did not make any difference.

**Comment by fenryxo** [comment]
Could you try to upgrade to [WebKitGTK+ 2.12.2](https://launchpad.net/~nuvola-player-builders/+archive/ubuntu/webkit)?

**Comment by wayneyooktz** [comment]
After adding the ppa and `sudo apt-get update/upgrade` the problem is still there.
The following packages were all successfully upgraded to version 2.12.2-2tiliado1~xenial1:
* libwebkit2gtk-4.0-dev
* libjavascriptcoregtk-4.0-dev
* libwebkit2gtk-4.0-37-gtk2
* libwebkit2gtk-4.0-37
* libjavascriptcoregtk-4.0-18
* gir1.2-webkit2-4.0
* gir1.2-javascriptcoregtk-4.0

**Comment by wayneyooktz** [comment]
Okay, I found out what caused the issue.
It was the GTK theme I was using. There's a theme called ["Flat-Remix-OS"](http://gnome-look.org/content/show.php/Flat-Remix-OS?content=175668) and when using it, WebKit behaves the way I said. I just switched to a different one, and everything is fine.
I'll try and contact the creator of the theme, so this issue can be closed.

Sorry for any inconvenience.
